### PR TITLE
Headless CMS: add custom links to block editor

### PR DESCRIPTION
**Pull request by jonasNobile**

### CODE REVIEW

File: headless-cms/admin/components/ContentField.tsx [hunk 1]
```diff
@@ -21,28 +24,53 @@ const inlineButtons: BlockEditorProps["inlineButtons"] = [
 		RB.headingOne,
 		RB.headingTwo,
 		RB.headingThree,
-		RB.headingFour,
+		RB.headingFour
```
Comment (VojtaStanek) [inline headless-cms/admin/components/ContentField.tsx:27]: I would leave trailing commas there.

File: headless-cms/package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-refresh": "^0.8.3",
+    "sass": "^1.49.0",
```
Comment (VojtaStanek) [inline headless-cms/package.json:27]: I would maybe remove this dependency and write it in pure CSS

**Review by VojtaStanek — CHANGES_REQUESTED** [review]

File: headless-cms/admin/components/ContentField.tsx (hunk 1 above)
Comment (jonasNobile) [reply headless-cms/admin/components/ContentField.tsx:27]: Ok, I'll add them back.

File: headless-cms/package.json (hunk 2 above)
Comment (jonasNobile) [reply headless-cms/package.json:27]: OK, I'll remove sass dependency.

**Review by VojtaStanek — APPROVED** [review]

**Pull request merged**